README.md

```diff
@@ -27,27 +27,37 @@ npm install
 
 
 ## Development Workflow
-Read the ["How to contribute and code of conduct"](CONTRIBUTING.md) documentation
-
-**Start a live-reload development server:**
 
+### Start a live-reload development server
 ```
-WEB_PATH="/" npm run dev
+npm run dev
 ```
+ 
+This will serve the LiMe App with hot reloading. By default it will proxy every backend request to http://10.13.0.1, the default ip address for LibreMesh nodes.
```

Comment:
You could mention that the app is a frontend for backend services running on the router,... that would explain the proxy part

Reply:
Yeah, good idea! Done